frontend/src/container/WorkspaceContent.jsx

```diff
@@ -749,9 +756,10 @@ class WorkspaceContent extends React.Component {
                     rootContentList={rootContentList}
                     isLast={isWorkspaceEmpty || isFilteredWorkspaceEmpty}
                     t={t}
-                  />}
+                  />
+                )}
 
-                {state.contentLoaded && (isWorkspaceEmpty || isFilteredWorkspaceEmpty)
+                {((state.contentLoaded && (isWorkspaceEmpty || isFilteredWorkspaceEmpty))
```

Comment:
I think there is one not required opening and closing parenthesis here

Reply:
I think it is clearer with the additional parentheses to clearly delimitate the condition of the ternary, but I can remove them if you think it is better without. What do you think?

Reply:
I'm fine with it, I thought it was an error